bundles/org.openhab.ui/web/src/pages/settings/things/channel/channel-general-settings.vue

```diff
@@ -2,20 +2,20 @@
   <f7-block class="padding-vertical no-padding-horizontal">
     <f7-col>
       <f7-list class="no-margin" inline-labels no-hairlines-md>
-        <f7-list-input v-if="createMode" :disabled="disabled" label="Channel Identifier" type="text" placeholder="Required" :value="channel.id"
+        <f7-list-input v-if="createMode" label="Channel ID" type="text" placeholder="Required" :value="channel.id"
                        info="Note: cannot be changed after the creation"
                        required validate pattern="[A-Za-z0-9_\-]+" error-message="Required. A-Z,a-z,0-9,_,- only"
                        @input="channel.id = $event.target.value" />
-        <f7-list-item v-if="!createMode" :disabled="disabled" media-item class="channel-item" title="Channel UID">
+        <f7-list-item v-else :disabled="disabled" media-item class="channel-item" title="Channel UID">
           <div slot="subtitle">
             {{ channel.uid }}
-            <clipboard-icon :value="channel.uid" tooltip="Copy UID" />
+            <clipboard-icon :value="channel.uid" tooltip="Copy UID" style="pointer-events: initial !important" />
           </div>
         </f7-list-item>
         <f7-list-input label="Label" type="text" :disabled="disabled" :placeholder="(channelType !== null) ? channelType.label : 'Required'" :value="channel.label" required validate
                        @input="channel.label = $event.target.value" :clear-button="disabled !== true" />
         <f7-list-input label="Description" type="text" :disabled="disabled" :placeholder="(channelType !== null) ? channelType.description : ''" :value="channel.description"
-                       @input="channel.description = $event.target.value" :clear-button="disabled !== true" />
+                       @input="channel.description = $event.target.value" :clear-button="!disabled !== true" />
```

Comment:
I think this change is wrong, I will revert it.

Reply:
I'm glad you caught this! I was just looking at `!disabled` didn't realise it was compared against true :(